jsk_pcl_ros/CMakeLists.txt

```diff
@@ -7,21 +7,46 @@ project(jsk_pcl_ros)
 set(PCL_MSGS pcl_msgs) ## hydro and later
 
 find_package(catkin REQUIRED COMPONENTS
-  jsk_recognition_utils jsk_pcl_ros_utils
-  dynamic_reconfigure pcl_ros nodelet message_generation genmsg
-  ${PCL_MSGS} sensor_msgs geometry_msgs jsk_recognition_msgs
-  eigen_conversions tf_conversions tf2_ros tf
-  image_transport nodelet cv_bridge
-  jsk_topic_tools
+  cv_bridge
+  diagnostic_msgs
+  diagnostic_updater
+  dynamic_reconfigure
+  eigen_conversions
+  genmsg
+  geometry_msgs
   image_geometry
-  jsk_footstep_msgs
+  image_transport
+  image_view2
   interactive_markers
+  jsk_footstep_msgs
+  jsk_pcl_ros_utils
+  jsk_recognition_msgs
+  jsk_recognition_utils
+  jsk_topic_tools
+  kdl_conversions
+  kdl_parser
   laser_assembler
-  octomap_server
-  octomap_ros
+  message_generation
+  ml_classifiers
+  moveit_core
+  nav_msgs
+  nodelet
   octomap_msgs
-  kdl_parser
-  kdl_conversions
+  octomap_ros
+  octomap_server
+  pcl_conversions
+  ${PCL_MSGS}
+  pcl_ros nodelet
+  rosboost_cfg
+  roscpp_tutorials
+  sensor_msgs
+  std_msgs
```

Comment:
it seems both tests passed build test, but failed in run_tests
https://travis-ci.org/jsk-ros-pkg/jsk_recognition/jobs/154928148, https://travis-ci.org/jsk-ros-pkg/jsk_recognition/jobs/154928146